packages/spanner/system-test/spanner.js

```diff
@@ -396,6 +396,10 @@ describe('Spanner', function() {
         done();
       });
     });
+
+    it('should keep the session alive', function(done) {
+      session.keepAlive(done);
```

Comment:
So sessions last an hour, I'm not really sure this is something we want to system test.